modules/exploits/linux/http/linksys_wvbr0_user_agent_exec_noauth.rb

```diff
@@ -0,0 +1,124 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
```

Comment:
Command injection that might be `ExcellentRanking`.

Reply:
heh, well if you say so ;)